src/base/projects/Blob.svelte

```diff
@@ -25,16 +26,23 @@
     ?.values()
     .next().value;
 
-  $: if (line) {
-    scrollIntoView(line);
-  }
+  // Waiting onMount, due to the line numbers still loading.
+  onMount(() => {
+    if (line) {
+      scrollIntoView(line);
+    }
+  });
```

Comment:
Strange, shouldn't the reactive statement below be enough?

Reply:
If we only keep the reactive statement, we don't take into account the delayed rendering of the line numbers.
But we need the reactive statemtent to handle the scrolling when a line number is clicked.